monkey/tests/mocked_data/server_configs/server_config_partial_credentials.json

```diff
@@ -3,5 +3,8 @@
     "server_config": "password",
     "deployment": "develop",
     "user": "test"
+  },
+  "mongodb": {
+        "start_mongodb": true
```

Comment:
These changes are not relevant to the PR, but I couldn't find a way to just commit the file move :D This will be added in the following PR anyways.